webuntis/objects.py

```diff
@@ -310,6 +310,26 @@ def code(self):
         if code in (None, u'cancelled', u'irregular'):
             return code
         return None
+        
+    @lazyproperty
+    def get_original_teacher(self):
+        ''' Support for original teacher '''        
+        try: 
+            return self._session.teachers(from_cache=True).filter(id=set([te[u'orgid'] for te in self._data[u'te']]))
```

Comment:
minor: You can drop the brackets here and use generator expressions instead of list comprehensions